### DISCUSSION TITLE
docker installation

### DISCUSSION THREAD

**Discussion by mustafa3161** [chatwoot/chatwoot#8901] [Self hosted]
i tried the guide provided for the docker installation and chatwoot itself opens and all but i don't seem to know how to get messages delivered to me, whether it's a configuration problem or something, i have no idea. would anyone be willing to help?!

**Comment by 170031284** [comment]
Hi Mustafa , 
check the docker logs using the command **docker-compose logs** and try to find the error logs and troubleshoot it . once you made any changes try to restart the docker using the command **docker-compoase-restart**.

Hope this helps

**Comment by mustafa3161** [comment]
`Attaching to root-base-1, root-sidekiq-1, root-rails-1, root-redis-1, root-postgres-1
root-base-1 | Switch to inspect mode.
root-base-1 | 
root-postgres-1 | 
root-postgres-1 | PostgreSQL Database directory appears to contain a database; Skipping initialization
root-postgres-1 | 
root-postgres-1 | 2024-02-11 11:09:05.465 UTC [1] LOG:  starting PostgreSQL 16.1 (Debian 16.1-1.pgdg120+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
root-postgres-1 | 2024-02-11 11:09:05.465 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
root-postgres-1 | 2024-02-11 11:09:05.466 UTC [1] LOG:  listening on IPv6 address "::", port 5432
root-postgres-1 | 2024-02-11 11:09:05.574 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
root-postgres-1 | 2024-02-11 11:09:05.628 UTC [29] LOG:  database system was interrupted; last known up at 2024-02-11 11:04:21 UTC
root-postgres-1 | 2024-02-11 11:09:06.056 UTC [29] LOG:  database system was not properly shut down; automatic recovery in progress
root-postgres-1 | 2024-02-11 11:09:06.072 UTC [29] LOG:  redo starts at 0/1C79538
root-postgres-1 | 2024-02-11 11:09:06.072 UTC [29] LOG:  invalid record length at 0/1C79620: expected at least 24, got 0
root-postgres-1 | 2024-02-11 11:09:06.072 UTC [29] LOG:  redo done at 0/1C795E8 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
root-postgres-1 | 2024-02-11 11:09:06.113 UTC [27] LOG:  checkpoint starting: end-of-recovery immediate wait
root-postgres-1 | 2024-02-11 11:09:06.328 UTC [27] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.063 s, sync=0.043 s, total=0.235 s; sync files=2, longest=0.026 s, average=0.022 s; distance=0 kB, estimate=0 kB; lsn=0/1C79620, redo lsn=0/1C79620
root-postgres-1 | 2024-02-11 11:09:06.346 UTC [1] LOG:  database system is ready to accept connections
root-sidekiq-1 | /gems/ruby/3.2.0/gems/gmail_xoauth-0.4.2/lib/gmail_xoauth/imap_xoauth_authenticator.rb:31: warning: Net::IMAP.add_authenticator is deprecated.  Use Net::IMAP::SASL.add_authenticator instead.
root-sidekiq-1 | /gems/ruby/3.2.0/gems/gmail_xoauth-0.4.2/lib/gmail_xoauth/imap_xoauth2_authenticator.rb:24: warning: Net::IMAP.add_authenticator is deprecated.  Use Net::IMAP::SASL.add_authenticator instead.
root-sidekiq-1 | fatal: not a git repository (or any of the parent directories): .git
root-sidekiq-1 | {"ts":"2024-02-11T11:09:38.618Z","pid":1,"tid":"5bp","lvl":"INFO","msg":"Sidekiq 7.2.0 connecting to Redis with options {:size=>10, :pool_name=>\"internal\", :url=>\"redis://redis:6379/1\", :password=>\"REDACTED\", :reconnect_attempts=>2, :timeout=>1}"}
root-sidekiq-1 | {"ts":"2024-02-11T11:09:38.650Z","pid":1,"tid":"5bp","lvl":"INFO","msg":"Cron Jobs - added job with name: internal_check_new_versions_job"}
root-sidekiq-1 | {"ts":"2024-02-11T11:09:38.665Z","pid":1,"tid":"5bp","lvl":"INFO","msg":"Cron Jobs - added job with name: trigger_scheduled_items_job"}
root-sidekiq-1 | {"ts":"2024-02-11T11:09:38.678Z","pid":1,"tid":"5bp","lvl":"INFO","msg":"Cron Jobs - added job with name: trigger_imap_email_inboxes_job"}
root-sidekiq-1 | {"ts":"2024-02-11T11:09:38.681Z","pid":1,"tid":"5bp","lvl":"INFO","msg":"Cron Jobs - added job with name: remove_stale_contact_inboxes_job.rb"}
root-redis-1 | 9:C 11 Feb 2024 11:09:05.377 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. Being disabled, it can also cause failures without low memory condition, see https://github.com/jemalloc/jemalloc/issues/1328. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
root-redis-1 | 9:C 11 Feb 2024 11:09:05.377 * oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
root-redis-1 | 9:C 11 Feb 2024 11:09:05.377 * Redis version=7.2.4, bits=64, commit=00000000, modified=0, pid=9, just started
root-redis-1 | 9:C 11 Feb 2024 11:09:05.377 * Configuration loaded
root-redis-1 | 9:M 11 Feb 2024 11:09:05.377 * monotonic clock: POSIX clock_gettime
root-redis-1 | 9:M 11 Feb 2024 11:09:05.378 * Running mode=standalone, port=6379.
root-redis-1 | 9:M 11 Feb 2024 11:09:05.380 * Server initialized
root-redis-1 | 9:M 11 Feb 2024 11:09:05.380 * Loading RDB produced by version 7.2.4
root-redis-1 | 9:M 11 Feb 2024 11:09:05.380 * RDB age 345 seconds
root-redis-1 | 9:M 11 Feb 2024 11:09:05.380 * RDB memory usage when created 2.10 Mb
root-redis-1 | 9:M 11 Feb 2024 11:09:05.381 * Done loading RDB, keys loaded: 53, keys expired: 2.
root-redis-1 | 9:M 11 Feb 2024 11:09:05.381 * DB loaded from disk: 0.001 seconds
root-redis-1 | 9:M 11 Feb 2024 11:09:05.381 * Ready to accept connections tcp
root-rails-1 | + rm -rf /app/tmp/pids/server.pid
root-rails-1 | + rm -rf '/app/tmp/cache/*'
root-rails-1 | + echo 'Waiting for postgres to become ready....'
root-rails-1 | Waiting for postgres to become ready....
root-rails-1 | + docker/entrypoints/helpers/pg_database_url.rb
root-rails-1 | + export 'POSTGRES_PORT=5432'
root-rails-1 | + PG_READY='pg_isready -h postgres -p 5432 -U postgres'
root-rails-1 | + pg_isready -h postgres -p 5432 -U postgres
root-rails-1 | postgres:5432 - accepting connections
root-rails-1 | Database ready to accept connections.
root-rails-1 | + echo 'Database ready to accept connections.'
root-rails-1 | + bundle install
root-rails-1 | Bundle complete! 125 Gemfile dependencies, 247 gems now installed.
root-rails-1 | Gems in the groups 'development' and 'test' were not installed.
root-rails-1 | Bundled gems are installed into `/gems`
root-rails-1 | + BUNDLE='bundle check'
root-rails-1 | + bundle check
root-rails-1 | The Gemfile's dependencies are satisfied
root-rails-1 | + exec bundle exec rails s -p 3000 -b 0.0.0.0
root-rails-1 | /gems/ruby/3.2.0/gems/gmail_xoauth-0.4.2/lib/gmail_xoauth/imap_xoauth_authenticator.rb:31: warning: Net::IMAP.add_authenticator is deprecated.  Use Net::IMAP::SASL.add_authenticator instead.
root-rails-1 | /gems/ruby/3.2.0/gems/gmail_xoauth-0.4.2/lib/gmail_xoauth/imap_xoauth2_authenticator.rb:24: warning: Net::IMAP.add_authenticator is deprecated.  Use Net::IMAP::SASL.add_authenticator instead.
root-rails-1 | => Booting Puma
root-rails-1 | => Rails 7.0.8 application starting in production 
root-rails-1 | => Run `bin/rails server --help` for more startup options
root-rails-1 | fatal: not a git repository (or any of the parent directories): .git`
those are the logs, not sure how to fix them
